src/styles/variables.scss

```diff
@@ -0,0 +1,16 @@
+$font-url: 'https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap';
+$card-width: 200px;
+$border-color: #f3f3f3;
+$padding: 16px;
+$border-radius: 5px;
+$background-color: #fff;
+$primary-color: #007bff;
+$hover-color: #00acdc;
+$main-color: #f5f5f5;
+$font-family: Roboto, Arial, Helvetica, sans-serif;
+$star-size: 16px;
+$star-image-default: url('../images/star.svg');
+$star-image-active: url('../images/star-active.svg');
+$min-height: 100vh;
+$buy-width: 166px;
```

Comment:
```suggestion
$buy-width: 100%;
```

Reply:
the layout specifies that the button size should be 166px